oauth/authentication-oauth2.0.md

```diff
@@ -1,6 +1,6 @@
 # Authorizing OAuth Apps
 
-OAuth2.0 is a protocol that allows SolveXia to communicate with third-party applications, authorise them and grant access via generation access tokens.
+OAuth2.0 is a protocol that allows SolveXia to communicate with third-party applications, authorise them and grant access via a generated access tokens.
```

Comment:
Should be "via a generated access token." (single) or "via generated access tokens." (plural)

Reply:
Yup, thanks @hal9000-hub, changed to "via a generated access token.".